### PR TITLE
Update and rename FunctionsAndModules.md to AutomaticFunctionsAndModu…

### PR DESCRIPTION
**Pull request by phatmandrake**
…les.md

I added an example of how to use an automatically generated route. The name of this section should also reflect what it's for.

### CODE REVIEW

File: docs/Tutorials/Routes/Utilities/AutomaticFunctionsAndModules.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ Add-PodeRoute -Method Post -Path '/Invoke-Expression' -ScriptBlock {
 }
 ```
 
+Example Route Invocation:
+
+```powershell
+Invoke-Restmethod http://localhost:8080/Invoke-Expression -Body @{Command = "Get-Date"}
+```
+Function parameters become hashtable key value pairs passed in the body of the request.
+
```
Comment (Badgerati) [inline docs/Tutorials/Routes/Utilities/AutomaticFunctionsAndModules.md:37]: "Invocation" should be a lowercase "i" :)

I actually just spotted that on line 19, `$WebEvent.Data` should be `$WebEvent.Query` for a GET call; as GET is via query string, and any other method is in the body payload. So might be worth showing both:

```powershell
Invoke-RestMethod -Uri 'http://localhost:8080/Invoke-Expression' -Body @{Command = "Get-Date"} -Method Post
Invoke-RestMethod -Uri 'http://localhost:8080/Get-ChildItem?Path=/some/path'
```

**Review by Badgerati — CHANGES_REQUESTED** [review]

**Comment by Badgerati** [comment]
Oh, just noticed, the PR should be against the develop branch.

File: docs/Tutorials/Routes/Utilities/AutomaticFunctionsAndModules.md (hunk 1 above)
Comment (phatmandrake) [reply docs/Tutorials/Routes/Utilities/AutomaticFunctionsAndModules.md]: Updated - also added an additional example for using the -body parameter for query parameters as well.

**Review by Badgerati — APPROVED** [review]

**Comment by Badgerati** [comment]
LGTM!

**Pull request merged**